packages/@sanity/types/src/schema/types.ts

```diff
@@ -31,6 +33,15 @@ export type SortOrdering = {
   }
 }
 
+export interface HiddenPredicateContext {
```

Comment:
Wondering if `HiddenCallbackContext` is more idiomatic to JS 🤔 See e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
I think we should include `path` here as well since the context in Rule.custom also includes path

Reply:
>Wondering if HiddenCallbackContext would be a more idiomatic naming

Yeah, lets follow the MDN terminology here. Thanks!